lib/any.ml

```diff
@@ -1,9 +1,29 @@
 module type Any = sig
-  type t_for_any
+  type t
+  val __any__ : unit
```

Comment:
Another option is to use a type instead of a value to "bless" the signatures, which has the advantage that it doesn't take up space at runtime:
```ocaml
# module type Zero = sig type t type zero end;;
[...]
# module type One = sig type t val one : unit end;;
[...]
# Obj.(size (repr (module struct type t type zero end  : Zero)));;
- : int = 0
# Obj.(size (repr (module struct type t let one = () end  : One)));;
- : int = 1
```